pkg/proxy/winkernel/proxier.go

```diff
@@ -1010,13 +1038,16 @@ func (proxier *Proxier) syncProxyRules() {
 		containsNodeIP := false
 
 		for _, epInfo := range proxier.endpointsMap[svcName] {
-
 			ep, ok := epInfo.(*endpointsInfo)
 			if !ok {
 				klog.Errorf("Failed to cast endpointsInfo %q", svcName.String())
 				continue
 			}
 
+			if !ep.IsReady() {
+				continue
+			}
+
```

Comment:
So, the behavior of this commit is that when using the winkernel proxier with EndpointSlice enabled, terminating pods will be ignored, whereas previously they would have been included?

Reply:
My reading of the PR is that this change is fixing a pre-existing bug in the winkernel proxier; previously it was including non-ready endpoints in its output if the `WindowsEndpointSliceProxying` feature gate was enabled (which it is not by default). With this fix, it now correctly ignores them.

It is not considering the terminating condition at all; this commit just brings winkernel up to parity with where the iptables and ipvs proxies were _before_ this PR.

Reply:
Understood, yay